packages/playwright/src/worker/testInfo.ts

```diff
@@ -361,6 +363,8 @@ export class TestInfoImpl implements TestInfo {
     const step = (error as any)[stepSymbol] as TestStepInternal | undefined;
     if (step && step.boxedStack)
       serialized.stack = `${error.name}: ${error.message}\n${stringifyStackFrames(step.boxedStack).join('\n')}`;
+    if (!retriable)
```

Comment:
It seems like this should be placed before the early return from this function.

Reply:
good point!